src/Behavior/BoolCast.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace ipl\Orm\Behavior;
+
+use InvalidArgumentException;
+use ipl\Orm\Contract\PropertyBehavior;
+
+use function ipl\Stdlib\get_php_type;
+
+/**
+ * Covert specific database values from and to boolean
+ *
+ * To unify the support of boolean values in different database systems,
+ * specific database values are converted to and from boolean values,
+ * e.g. by default `n` is converted to `false` and `y` to `true` and vice versa respectively,
+ * which could be stored as `ENUM('n', 'y')`.
```

Comment:
> ENUM('n', 'y')

Is that the sort-safe order already? :sweat_smile:

Reply:
Yes.